cluster-autoscaler/README.md

```diff
@@ -0,0 +1,48 @@
+# CLUSTER AUTOSCALER
+
+## Synopsis 
+
+We are spinning a 3 node cluster on Azure or GCP.
+A specfication for the autoscaler is defined in the yaml file.
+The cluster autoscaler will look out for pending nodes and then scale up based on the VM flavour defined in the autoscaler yaml
+A metrics tools will handle logs and metrics of the cluster for the different autoscaling iterations
+
+
+## Context
+
+We are using kubernetes cluster autoscaler
+
+
+## What we are using
++2 node cluster
+Master with enabled autoscaler
+GCP/Azure dashboard to handle metrics
+
+
+## Architecture
+[TO DO]
+
+
+## State diagram
+
+![Alt text](https://github.com/fahedouch/kubernetes-workshop/blob/cluster_autoscaler_initial_commit/cluster-autoscaler/state-diagram.png "CA state diagram")
```

Comment:
3 nodes granted? 
So instead of 1 pod say pods in pending state 
Sequence be like: 4 pods gets triggered, few gets schedule , few in pending, CA is running , see pending pods, request for node, node get added.

Reply:
``3 nodes granted?``
it is step 3 and not the number of node
Same thing ``1`` is for step 

``1`` ``2`` ``3`` ``4`` are steps in state diagram and not number ;)

I made some modifications to make this explicit